files/init_db.sh

```diff
@@ -39,5 +39,5 @@ wait
 # we can swap it in now
 touch ~/clean
 
-# continue dumping the ONSUD table in the background
-pg_dump -d polling_stations -t addressbase_onsud > /dev/null
+# continue dumping the gss to council lookup table in the background
+pg_dump -d polling_stations -t addressbase_uprntocouncil > /dev/null
```

Comment:
Well done for thinking of this. It was top of my list of non-obvious things you might not have thought of :+1: 

Given this table is going to be loooooads smaller than the ONSUD, you can probably get away with doing this in parallel with the other ones. The reason I put this here was because it was the slowest table to dump and pre-warming the others was enough to get performance acceptable , but if it takes less time than `addressbase_address` we might as well move it. If you want, we can run through this in the next couple of weeks. For now, it won't do any harm to leave it here.